.gitignore

```diff
@@ -2,3 +2,4 @@
 /kubectl
 vendor/
 tmp/
+.idea
```

Comment:
You shouldn't add this to every project. Please use a global gitignore on your machine like:
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Done